templates/assets.html

```diff
@@ -65,6 +65,17 @@ <h3 class="asset-subsection" id="{{ section.title | slugify }}">
 
         {% endfor %}
         {% endfor %}
+        <div id="suggestions-footer">
+            <div class="media-content">
+                Didn't find what you were looking for? How about searching our other community resources?
```

Comment:
```suggestion
                Didn't find what you were looking for? See what our other community resources have on your search query.
```
I feel like it wasn't exactly obvious that the links would already contain the search query; my intuition was that they'd just take me to the site with no query. Should clarify wording imo.

Reply:
```suggestion
                Didn't find what you were looking for? See what your search reveals in our docs and in other corners of the community.
```